rllib/algorithms/dt/dt.py

```diff
@@ -78,6 +78,15 @@ def __init__(self, algo_class=None):
             # Do not change the type of replay buffer.
             "type": MultiAgentSegmentationBuffer,
         }
+        self.exploration_config = {
```

Comment:
This is offline RL, it should not need `self.exploration_config`'s default of stochasting sampling.